package.json

```diff
@@ -2,10 +2,9 @@
   "name": "daf",
   "private": true,
   "scripts": {
-    "build": "yarn lerna run build && yarn extract-api && yarn generate-plugin-schema && yarn docs:api && yarn docs:examples && yarn docs:clean-temp && yarn prettier:examples",
```

Comment:
@jasonhealy why did you remove `yarn extract-api` and `yarn docs:examples`?
These are needed to extract inline doc examples and include them into our integration tests